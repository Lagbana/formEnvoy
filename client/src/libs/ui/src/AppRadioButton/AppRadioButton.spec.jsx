import React from 'react'
import { shallow, configure } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'
import { AppRadioButton } from '.'

configure({ adapter: new Adapter() })

jest.mock('@react-navigation/native')

test('<AppRadioButton /> renders correctly', () => {
  const component = shallow(
    <AppRadioButton
			options={[]}
			onSelect={() => {}}
    />
  )
  expect(toJson(component)).toMatchSnapshot()
})
