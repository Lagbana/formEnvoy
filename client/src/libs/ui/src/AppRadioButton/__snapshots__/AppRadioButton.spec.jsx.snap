// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppRadioButton /> renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "wrap",
        "justifyContent": "center",
        "marginHorizontal": -7,
        "marginVertical": 20,
      },
      undefined,
    ]
  }
/>
`;
